src/modules/facets/ui.js

```diff
@@ -29,16 +29,17 @@ function selectedFacetItemTemplateUI (selectedFacet,selectedFacetItem){
                 `<button class="UNX-delete-facet ${removeFacetsSelectorClass} ${css}" data-id="${dataId}" data-facet-action="${action}" data-facet-name="${facetName}">x</button></div>`
             ].join('');
 }
-function selectedFacetUI(selections, facet) {
+function selectedFacetUI(selections, facet,selectedFacetsConfig) {
     const {
         clearAllText,
         clearFacetsSelectorClass
     } = facet;
+    const selectedFClass = (this.selectedFacetClass)?this.selectedFacetClass:selectedFacetsConfig.selectedFacetClass;
```

Comment:
Same as above. Priority must be given to `selectedFacetsConfig.selectedFacetClass` before the deprecated config.